tests/test_html.py

```diff
@@ -0,0 +1,14 @@
+import tempfile
+
+from click.testing import CliRunner
+
+from newshomepages import html
+
+TEMP_DIR = tempfile.gettempdir()
+
+
+def test_single():
+    """Test a single HTML save."""
+    runner = CliRunner()
+    result = runner.invoke(html.cli, ["latimes", "-o", TEMP_DIR])
+    assert result.exit_code == 0
```

Comment:
Can we use the temporary path provided by pytest? 

https://docs.pytest.org/en/6.2.x/tmpdir.html